hw/snitch_cluster/src/snitch_cluster_pkg.sv.tpl

```diff
@@ -93,6 +96,13 @@ package ${cfg['cluster']['name']}_pkg;
   `AXI_TYPEDEF_ALL(narrow_out, addr_t, narrow_out_id_t, data_t, strb_t, user_t)
   `AXI_TYPEDEF_ALL(wide_in, addr_t, wide_in_id_t, data_dma_t, strb_dma_t, user_dma_t)
   `AXI_TYPEDEF_ALL(wide_out, addr_t, wide_out_id_t, data_dma_t, strb_dma_t, user_dma_t)
+  `AXI_TYPEDEF_ALL(narrow_ext, addr_t, narrow_out_id_t, data_t, strb_t, user_t)
+
+  typedef logic [TcdmAddrWidth-1:0]     tcdm_addr_t;
+  typedef logic [WideDataWidth-1:0]     data_ext_t;
+  typedef logic [WideDataWidth/8-1:0]   strb_ext_t;
+
+  `TCDM_TYPEDEF_ALL(tcdm_ext, tcdm_addr_t, data_ext_t, strb_ext_t, logic)
 
```

Comment:
Similar types are also defined in `snitch_cluster.sv`. I see the need to provide them to the external user, but I think we can avoid duplicating them. Perhaps we can just move the existing ones to the package.